### DISCUSSION TITLE
Improve SVG export

### DISCUSSION THREAD

**Discussion by merco** [UglyToad/PdfPig#378] [Ideas]
would it be possible to improve the export of SVG?
Currently the file is not even readable because the namespace in the svg tag is missing ... 
but if I add it ... I find that probably the texts are positioned correctly but not the paths.
It would also be interesting to compress these paths to be able to indicate the number of decimals and / or Ramer – Douglas – Peucker algorithm to reduce the number of points?

![immagine](https://user-images.githubusercontent.com/1032335/137104894-9103e3a3-8489-4eda-b090-cce1c1d6c332.png)

thank you

**Comment by BobLd** [comment]
Hi @merco,

Thanks a lot for your question.

How did you export to svg? Did you use `SvgTextExporter`? I wrote this class a while ago and it might need an update... As far as I remember, it used to work well with paths, but not with characters (problem was that the letter's path was not always available)...

Regarding Ramer-Douglas-Peucker, this is a very good point. I did 2 repos on this algo, maybe it's a good entry point if you want to help with that:
- https://github.com/BobLd/RamerDouglasPeuckerNet
- https://github.com/BobLd/RamerDouglasPeuckerNetV2

Unfortunatly I have limited time at hand, but happy to help if you want to contribute

**Reply by merco** [reply]
Yes ...SvgTextExporter, but I think chars are ok... the problems are with the paths.
